prow/config.gen.yaml

```diff
@@ -309,21 +309,16 @@ postsubmits:
         imagePullPolicy: Always
         command:
         - ./maistra/ci/post-submit.sh
+        env:
+        - name: BAZEL_REMOTE_CACHE
+          value: http://bazel-cache.default/cache
```

Comment:
Once this PR merges, I'll update the envoy&proxy scripts to make use of this variable.